ark/phenotyping/run_cell_som.R

```diff
@@ -13,48 +14,53 @@ library(FlowSOM)
 # get the command line arguments
 args <- commandArgs(trailingOnly=TRUE)
 
-# get the path to the cluster counts data
+# get the path to the cluster counts data (norm)
 clusterCountsPath <- args[1]
 
 # get the weights write path
 cellWeightsPath <- args[2]
 
-# get the cluster write path
-cellClusterPath <- args[3]
+# get the cluster write path (normalized)
+cellClusterPathNorm <- args[3]
 
-# read the cluster counts data
+# read the cluster counts data (norm)
 print("Reading the cluster counts data")
-clusterCountsData <- arrow::read_feather(clusterCountsPath)
+clusterCountsNorm <- as.data.frame(arrow::read_feather(clusterCountsPath))
 
 # read the weights
 print("Reading the weights matrix")
 somWeights <- as.matrix(arrow::read_feather(cellWeightsPath))
 
-# get the column names of the SOM pixel/meta clusters
-clusterCols <- colnames(clusterCountsData)[grepl("cluster_|hCluster_cap_",
-                                           colnames(clusterCountsData))]
+# define the subset of count columns to cluster on
+clusterCols <- colnames(clusterCountsNorm)[grepl("cluster_|hCluster_cap_",
+                                           colnames(clusterCountsNorm))]
```

Comment:
I think the column names in the grepl need to be changed after you renamed them.